template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+
+- **Looking at lots of examples.** Start by searching for examples of that type of document they want to create a template for. The more examples you can look at, the better. While it’s better to review good examples of that type of document, there is actually a lot of value in reviewing bad examples too. Consider keeping a spreadsheet to track which examples you used, what elements each one had in common, and what you thought was effective or ineffective.
+- **Searching for guides, books, blog posts, conference presentations, or videos about best practices.** Search the Internet to find advice, tips, or expert research about how to create that type of document. Consider posting in a forum for resource ideas. For example, asking for helpful guides or insights on a community forum like the Write the Docs Slack workspace could be beneficial. Be mindful of, and respect copyright terms of source material. Do not plagiarize and offer attribution where appropriate.
+- **Reaching out to experts.** When you find people you admire, who have researched your topic already, try reaching out to them. They often have a “how to contact me” webpage. Ask if they’d be okay with their material being used. (They might need to republish under a different copyright.) Invite them to participate in the template working group. They might even be prepared to lead it. If you feel shy about reaching out yourself, your template mentor or senior Good Docs Project member might offer to help.
+- **Collaborating with others in a working group.** Consider joining one of the template-focused working groups to meet regularly with a workshop of writers and discuss research ideas and findings.
+
+
+## Workshop the draft of the template
+
+In this phase, you’ll begin to share your draft with community reviewers and invite feedback.
+Optionally, you might also consider sharing it beyond our community with other technical writing communities such as Write the Docs or beyond.
+The workshop phase is arguably the most crucial and important phase in the template writing process, so your template project might spend the bulk of its time in this phase.
+
+To share your Google Docs drafts:
+
+1. Inside the draft, click the **Share** button and change the **Get Link** settings to: **Anyone on the internet with this link can create comments.**
+
+2. Copy the link to your Google Doc drafts into the issue that corresponds with your template in the templates repository.
+
+3. Notify the templates coordinator or your template mentor, who will help you get reviews for your draft from the community reviewers group.
+
+The templates coordinator will help ensure you get at least three high quality reviews from the community reviewers. When you’ve received sufficient community input and incorporated suggestions into your draft, you will collaborate with a templates coordinator to determine that your draft is ready to move to the next phase.

```

Comment:
I'm a little worried about the level of coordination that might be required here, especially when considering iterations on templates post-v1, where each minor change requested might be represented by an issue, each of which would need the required three reviews.

Reply:
Maybe two reviewers needed for a new template, and one reviewer needed for an update to an existing template?

Reply:
Keep in mind that we already have a requirement to author templates as part of a template-focused working group. So if you could even just get three community reviewers from your group, that satisfies the requirement. Our two template-focused working group currently both have well above three members. And then you always have the larger community to draw upon, of which we have many more than three.

Reply:
In last week's general meeting, we decided that, for now, we'll keep the requirement to have at least three community reviews are required to move to the next phase. We'll adjust this requirement if we discover it is too difficult.

Reply:
My memory of this meeting was that we wouldn't make "three" a hard requirement, but rather a strong recommendation.
```suggestion
The templates coordinator will help you find reviewers. Typically three or more community reviewers will provide a detailed review of your material. When you’ve received sufficient community input and incorporated suggestions into your draft, you will collaborate with a templates coordinator to determine that your draft is ready to move to the next phase.
```